eShopSolution.Appliaction/Manage/ManageProductService.cs

```diff
@@ -0,0 +1,241 @@
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+using eShopSolution.ViewModels.Common;
+using Microsoft.AspNetCore.Http;
+using System.Net.Http.Headers;
+using System.IO;
+using eShopSolution.Appliaction.Common;
+using eShopSolution.Appliaction.Interface;
+using eShopSolution.ViewModels.ViewModel;
+using eShopSolution.ViewModels.Request;
+
+namespace eShopSolution.Appliaction.Manage
+{
+    public class ManageProductService : IManageProductService
+    {
+
+        private readonly EShopDBContext _context;
+        private readonly IStorageService _storageService;
+        public ManageProductService(EShopDBContext context , IStorageService storageService)
+        {
+            _context = context;
+            _storageService = storageService;
+        }
+
+        //public async Task<int> AddImages(int productId, List<IFormFile> files)
+        //{
+        //    throw new NotImplementedException();
+        //}
+
+        public async Task AddViewCount(int productId)
+        {
+            var product =await _context.Products.FindAsync(productId);
+            product.ViewCount += 1;
+             await _context.SaveChangesAsync();
+        }
+
+        public async Task<int> create(ProductCreateRequest request)
+        {
+            var product = new eShopSolution.Data.Entities.Product()
+            {
+                Price = request.Price,
+                OriginalPrice = request.OriginalPrice,
+                Stock = request.Stock,
+                ViewCount = 0,
+                DateCreated = DateTime.Now,
+                ProductTranslations = new List<ProductTranslation>()
+                {
+                    new ProductTranslation()
+                    {
+                        Name = request.Name,
+                        Description = request.Description,
+                        Details = request.Details,
+                        SeoAlias = request.SeoAlias,
+                        SeoTitle = request.SeoTitle,
+                        SeoDescription = request.SeoDescription,
+                        LanguageId = request.LanguageId
+                    }
+                }
+            };
+            //Save Image
+            if (request.ThumbnailImage != null)
+            {
+                product.ProductImages = new List<ProductImage>()
+                {
+                    new ProductImage()
+                    {
+                        Caption = "Thumbnail image",
+                        DateCreated = DateTime.Now,
+                        FileSize = request.ThumbnailImage.Length,
+                        ImagePath = await this.SaveFile(request.ThumbnailImage),
+                        IsDefault = true,
+                        SortOrder = 1
+                    }
+                };
+            }
+            _context.Products.Add(product);
+            await _context.SaveChangesAsync();
+            return product.Id;
+        }
+
+      
+
+        public async Task<int> delete(int productId)
```

Comment:
Please follow the naming convention rules for function name, variable name etc..